tests/test_overdrive.py

```diff
@@ -1093,6 +1094,98 @@ def test_sync_bookshelf_ignores_holds_from_other_collections(self):
         eq_(5, len(patron.holds))
         assert overdrive_hold in patron.holds
 
+class TestOverdriveCircluationMonitor(OverdriveAPITest):
```

Comment:
We were simply missing this test coverage and I had to write a big test to make sure my change didn't break anything.